os/board/rtl8721csm/src/component/common/api/example_eap.c

```diff
@@ -0,0 +1,238 @@
+"Rm5RQfCu69aGziqJUjeCKLNNo/flCz93yjeukV0jMRQrt4gdXzNLqsuuLQiRYMBE\r\n" \
+"8xURdKMZk68/c3mZdn77wjs2eVYX5PhUuuYCIPLNmu0ut+lmTQ17mFsINrT1sVFx\r\n" \
+"W/+4RLEzZzoFxG6BlRgXxKvknrRto0z8p7RmxejX/qq68G6uCEZToEsCAwEAAaNP\r\n" \
+"ME0wEwYDVR0lBAwwCgYIKwYBBQUHAwIwNgYDVR0fBC8wLTAroCmgJ4YlaHR0cDov\r\n" \
+"L3d3dy5leGFtcGxlLmNvbS9leGFtcGxlX2NhLmNybDANBgkqhkiG9w0BAQsFAAOC\r\n" \
+"AQEAQvW/IVSm3wiatOmsdjrQ58bo8QBsp0J+7CaRMRqjnVI6v12TSDRwpYQ/JTJg\r\n" \
+"fu+4BlKB+7NKQeQU01uNXXE53tPpE6MDr9MWwp4RUAsbvFdrt6jUvl5VBF9+st9E\r\n" \
+"9Sy5mZZ8pn5qtKnqtK5FvcNs0Ku3+YhKyTWIBTSOzhkwynb8+QfkIOoVfA1uxcbu\r\n" \
+"AtUtRn3tjXMt+EGCTfqS++Bwm+qFGl7HEgif9Fhy+1qics2iaOkxGm3ZdU1hqR+S\r\n" \
+"gkcpYmF6HI/4b5JbiaKCYeO8LIxJFOsVhotDwYuxCo6N1HNpEjW6MYMY8LtqrkJ+\r\n" \
+"6rmIGg84Yxo0wr4ILpnviyP/vw==\r\n" \
+"-----END CERTIFICATE-----\r\n";
+	eap_client_key = \
+"-----BEGIN RSA PRIVATE KEY-----\r\n" \
+"MIIEowIBAAKCAQEAuZVgiTvd5LLhXksTjcnSYVQGEXwGFWfzqJOJayqnLB0p1fBu\r\n" \
+"KuFKnBFsah4LRJ+T28Jg6nqi2xCdIl1SB7CTpUV3l6aSu6crd5bK0BKHhNG52ebI\r\n" \
+"LIUe3ko9XK9LalAFDCrLVBYubqVsCIe+oqeFjoCe4sByefRGblFB8K7r1obOKolS\r\n" \
+"N4Ios02j9+ULP3fKN66RXSMxFCu3iB1fM0uqy64tCJFgwETzFRF0oxmTrz9zeZl2\r\n" \
+"fvvCOzZ5Vhfk+FS65gIg8s2a7S636WZNDXuYWwg2tPWxUXFb/7hEsTNnOgXEboGV\r\n" \
+"GBfEq+SetG2jTPyntGbF6Nf+qrrwbq4IRlOgSwIDAQABAoIBACckoaLwdKHrNOvN\r\n" \
+"jAdI/U24Cjuf3Cyhq1vwGnj3GyGL2Sodty7mK/V1G1falGfZ/LbKbeSnDziDbZE0\r\n" \
+"+7tRfCLM3yemY6ae4OUtcgOnFNxn2/pg2fHB9jvOjtjAxHhc/mONlytaLlr6GypF\r\n" \
+"b2khSmM/L8QF8V9VrZFRJEZerrGliza8RfkptdfuV3UzGyhuGK8NfV6sYgmt3oiE\r\n" \
+"uWfSlui+zF5BoXWZ5WEPPAUn5pSUgqTzOa3AxrdQFWudNmXJmUYxbkNtmoS+HRAl\r\n" \
+"GwD/2a3N4mEY9b490p4r6TtLhkyspD8VG/1xlzgPNTbHKztzZ7YLHzJ1RSn1iXju\r\n" \
+"uitteFkCgYEA5uqPCD1kawai5zkyptULVrCqmaUzmq/EwpHY/hdiWRGZPjTPOGcK\r\n" \
+"325nTtbfGUf3X6RyVfsEkwh41TxquEOy69hE5+fa8mw/1zGE7M2tdP+dGVO4OSmd\r\n" \
+"pIculTmyzHsrq5VRl7twz2h6v7OzV172hYsEB2NrfNXpsC7AkjKhRgcCgYEAzb4u\r\n" \
+"dv2fTkQB/uNIGaQgdlI2ULjPlys7rArIR3eoq+AXBggTuW3G/qZ8kZz3m5nD125L\r\n" \
+"xCOK8hS+CO2Rp2ZPyIBh7hO10kvq5k52VBRqeF6hZKBGgo9cbtFLMSOX6POBRvyJ\r\n" \
+"CCYU+OZTKSNaTtgtX8AaeawQ7vbvvsTJDJ87Yp0CgYAlE8aRoSHtmWwwyltatFyU\r\n" \
+"t9s8vmzxyoVCZJ699c1B1fSQK3SWbt0tERIfc5rw+UA8KDB/o8WRQkv8mS3dlmk1\r\n" \
+"O02dGvpp8H2ztOzOBRxz69MyzOGItCju0RHmvhP6h1B0xY4LWfp8Ix+KhnUmc0YM\r\n" \
+"HU0xToJk0ggq1Obw0pZYowKBgQC4J1Yno3OH9k9i+cR7TvDPd16uef01bbkHkM8b\r\n" \
+"SPtsfDOfbnNZMe0/2Xz+SOVIHSWyGkMu2DFx5ScDjsEkeOtIp8D//Bxukw05/1jigT\r\n" \
+"Obcfg7vUnfarptZmBLJ3S9SdeY1klBRfTXpDEjTUVz9QMIBUatSVoOynBk7HaYGZ\r\n" \
+"5p8QKBgBRs15By+YfLC7M2VvsoJi191yJHRQDenM/y9pE9S/kACN6+VU93wj+EfE\r\n" \
+"/CfKt+sNBppc7o2cMAvHN0qcXtdjiLZfTRnO4yB38SlHLIAG4lL/vCMdXyVCklhU\r\n" \
+"cpAfNFNvD7S690qZg51fbGGbJi1AkrtM9rH7UcWzIFry2VM5GB\r\n" \
+"-----END RSA PRIVATE KEY-----\r\n";
+
+/*
+	Verify server's certificate is an optional feature.
+	If you want to use it please make sure ENABLE_EAP_SSL_VERIFY_SERVER in platform_opts.h is set to 1,
+	and the eap_ca_cert is set correctly.
+*/
+/*	eap_ca_cert = \

```

Comment:
Could you please avoid keeping commented lines in code that are not description comments?